python-minimal/README.md

```diff
@@ -0,0 +1,46 @@
+# Minimal python in Alpine image
+
```

Comment:
would be good to explain why this is needed and where it's used

Reply:
Done, but seems like that would be kind of obvious as its in the dp-concourse-tools repo ...